### ISSUE TITLE
v0.11.0 of prettycron

### ISSUE THREAD

**Issue by extend1994** [azza-bazoo/prettycron#33]
Hi
I am a member of CDNJS and the one who is in charge of adding prettycron to CDNJS according to the request cdnjs/cdnjs#10567.
I found that there is v0.11.0 on GitHub, is it possible for you to release it on npm?
It will help me use npm as the source to let prettycron be more compete on CDNJS :smile:
Thank you!

**Comment by r0bnet** [comment]
NPM package is indeed outdated. I stumbled across this issue today because the NPM package does not support the seconds argument in the toString() method but the Github version does. So it would be nice if you could update the package accordingly.

**Comment by zentrope** [comment]
Yes, I have to use a github URL in package.json to pull down the latest version (which has the fixes I need).

**Comment by extend1994** [comment]
@r0bnet @zentrope thanks!
Then is it possible to release v0.9.0 and v0.9.1 on GitHub? Or you don't need me to add these two versions to our CDN? Sorry for my bad english :disappointed:

**Comment by zentrope** [comment]
@extend1994 I'm not sure what you mean, but I have to add this:

```javascript
"dependencies" : {
  "prettycron": "https://github.com/azza-bazoo/prettycron.git#d0b633ee482d1d7e0fe9853578c72af541b815e3"
}
```

to my `package.json` file to get the most recent commits. Not ideal for caching in some local/corporate artifactory, if that's what you mean.

Ideally, someone will publish this to the NPM registry (or whatever one does -- I'm not a node.js dev by trade).

**Comment by r0bnet** [comment]
@extend1994 I think 0.9.0 stills has some bugs so it does not necessarily need to be published to your CDN. But 0.11.0 works fine for me so it would be nice to see this version at least on NPM. I can't decide it for your CDN.

**Comment by toniov** [comment]
It would be nice if 0.11.0 is published to NPM. 
I had to add a commit to the `package.json` as well.

**Comment by jakubknejzlik** [comment]
Or at least publish it as `0.11.0-prerelease` so we can install it by `prettycron@0.11.0-prerelease`.

**Comment by OskarPersson** [comment]
Any updates here?
